classes/patterns/class-issues.php

```diff
@@ -28,16 +28,20 @@ public static function get_name(): string {
 	 */
 	public static function get_media_text_template(): string {
 		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-40x40.jpg';
-		return '
-			<!-- wp:media-text {"mediaWidth":15,"mediaLink":"' . $media_link . '","mediaType":"image","mediaSizeSlug":"thumbnail","isStackedOnMobile":false,"verticalAlignment":"center","imageFill":false,"backgroundColor":"white","className":"is-style-large-padding force-no-lightbox"} -->
-			<div class="wp-block-media-text alignwide is-vertically-aligned-center has-white-background-color is-style-large-padding has-background force-no-lightbox" style="grid-template-columns:15% auto">
-			<figure class="wp-block-media-text__media"><img src="' . $media_link . '" alt="' . __( 'Default image', 'planet4-blocks-backend' ) . '"/></figure>
+
+		return '<!-- wp:group {"className":"d-flex jujstify-content-center has-white-background-color has-background","style":{"spacing":{"padding":{"top":"32px","right":"16px","bottom":"32px","left":"16px"}}},"backgroundColor":"white"} -->
```

Comment:
I doubt `jujstify-content-center` is what you meant to write here 😛

Reply:
Good catch! Thanks :)